packages/cli/package.json

```diff
@@ -1,39 +1,75 @@
 {
   "name": "@wcfactory/cli",
   "description": "CLI for managing your custom elements at scale.",
-  "main": "./bin/wcfactory-cli.js",
+  "version": "0.1.8",
+  "author": "heyMP @heyMP",
+  "bin": {
+    "wcfactory": "./bin/run"
+  },
+  "bugs": "https://github.com/elmsln/wcfactory/issues",
+  "dependencies": {
+    "@oclif/command": "^1",
+    "@oclif/config": "^1",
+    "@oclif/plugin-help": "^2",
+    "@wcfactory/common": "^0.1.8",
+    "@wcfactory/generator-wcfactory": "^0.1.8",
+    "inquirer": "^6.2.0",
+    "inquirer-recursive": "^0.0.5",
+    "listr": "^0.14.2",
+    "listr-update-renderer": "^0.4.0",
+    "listr-verbose-renderer": "^0.4.1",
+    "lodash": "^4.17.11",
+    "tslib": "^1",
+    "yeoman-environment": "^2.3.3",
+    "yeoman-generator": "^3.1.1"
+  },
+  "devDependencies": {
+    "@oclif/dev-cli": "^1",
+    "@oclif/test": "^1",
+    "@oclif/tslint": "^3",
+    "@types/chai": "^4",
+    "@types/mocha": "^5",
+    "@types/node": "^10",
+    "chai": "^4",
+    "globby": "^8",
+    "mocha": "^5",
+    "nyc": "^13",
+    "ts-node": "^7",
+    "tslint": "^5",
+    "typescript": "^3.0"
+  },
   "engines": {
-    "node": ">=6.0"
+    "node": ">=8.0.0"
   },
+  "files": [
+    "/bin",
+    "/lib",
+    "/npm-shrinkwrap.json",
+    "/oclif.manifest.json"
+  ],
+  "homepage": "https://github.com/elmsln/wcfactory",
   "keywords": [
-    "web",
-    "component",
-    "custom",
-    "element",
-    "cli"
+    "oclif"
   ],
-  "author": "",
   "license": "Apache-2.0",
-  "dependencies": {
-    "@wcfactory/generator-wcfactory": "^0.1.8",
-    "commander": "^2.18.0",
-    "please-upgrade-node": "^3.1.1",
-    "update-notifier": "^2.5.0",
-    "yeoman-environment": "^2.3.3"
+  "main": "lib/index.js",
+  "oclif": {
```

Comment:
Interesting, this is the first time I've heard of Open CLI.
Most CLI project's I've seen use http://yargs.js.org, and more recently there has been an uptick in the use of https://www.npmjs.com/package/ink

How are you liking Open CLI so far?

Reply:
oooo I have not seen ink.  That definitely looks interesting.  Open CLI is good but it's not doing a whole lot.  It's one level above yargs but in my opinion I would prefer something like ink.  Ink looks like we can customize the output to a much greater extent.

In this pull request I have Open CLI, Inquirer.js, Listr smooshed together.  I'm finding that Inquirer and Listr get me 80% there but it's going to be tough to get the next 20%.

What I will probably do is finish this pull request with Open CLI and remove any of the stuff that isn't fully working.  Then we can take a look at Ink or something similar to give us full control over the output.